core/Objects/ProjectConfig/ProjectConfigV1.cs

```diff
@@ -64,6 +65,7 @@ public class ProjectConfigV1
     public Uri DeliveredRepo { get; set; }
     [Newtonsoft.Json.JsonConverter(typeof(UriConverter))]
     public Uri CIRepo { get; set; }
+    public List<Uri> ReleaseRepos { get; set; }
```

Comment:
I'm not sure we need this field, this information seems duplicated from repositories.json which already supported multiple entries. I'd prefer referring to that file and remove the repo information from the project config altogether.

My reasoning is that adding this new field will force us to migrate the project config file in all future CLI releases. Any already scaffolded project will need to update this file. I'd avoid this unless absolutely necessary. I also find confusing having the same information in 2 separate files.